client/state/ui/guided-tours/contexts.js

```diff
@@ -62,8 +62,18 @@ const timeSinceUserRegistration = state => {
  * @return {Boolean} True if user is new, false otherwise
  */
 export const isNewUser = state => {
+	return isUserNewerThan( state, WEEK_IN_MILLISECONDS );
```

Comment:
I had the same thought, but not sure how to mock `Date.now()` in `timeSinceUserRegistration`?